doc/tools/coqrst/notations/html.py

```diff
@@ -0,0 +1,65 @@
+##########################################################################
+##         #   The Coq Proof Assistant / The Coq Development Team       ##
+##  v      #   INRIA, CNRS and contributors - Copyright 1999-2018       ##
+## <O___,, #       (see CREDITS file for the list of authors)           ##
+##   \VV/  ###############################################################
+##    //   #    This file is distributed under the terms of the         ##
+##         #     GNU Lesser General Public License Version 2.1          ##
+##         #     (see LICENSE file for the text of the license)         ##
+##########################################################################
+"""A visitor for ANDTLR notation ASTs, producing raw HTML.
```

Comment:
woops, there's a typo here ("ANDTLR")

Reply:
There seem to be 4 occurrences of the typo :)